tests/v2/test_message_store.nim

```diff
@@ -13,6 +13,7 @@ suite "Message Store":
       database = SqliteDatabase.init("", inMemory = true)[]
       store = WakuMessageStore.init(database)[]
       topic = ContentTopic("/waku/2/default-content/proto")
+      pubsubTopic = "pubsubTopic"
```

Comment:
I'd probably use default topic here just for consistently

Reply:
Do you mean default pubsub topic? do we have a default pubsub topic for the store protocol?

Reply:
Oh, I think you mean "/waku/2/default-waku/proto"

Reply:
Yes, good idea!